app/helpers/datasets_helper.rb

```diff
@@ -8,6 +8,22 @@ def dataset_field(f, dataset, options)
   }
 
   def friendly_frequency(frequency)
-    FRIENDLY_FREQUENCIES.fetch(frequency, 'One-off')
+    FRIENDLY_FREQUENCIES.fetch(frequency, frequency.humanize)
+  end
```

Comment:
Probably ok for now, but humanize won't do it as soon as we have content changes. It also doesn't work with internationalisation/localisation.